packages/@lightningjs/ui-components/src/components/ListItem/ListItemPicker.mdx

```diff
@@ -52,14 +53,18 @@ class Basic extends lng.Component {
 
 ## API
 
-### Properties
+### Parent Properties
+
+`ListItemPicker` has the same properties as `ListItem`
 
-`ListItemPicker` has the same properties as the ones listed in [ListItem](?path=/docs/components-listitem--list-item) in addition to a couple listed below:
+<ArgTypes
+  of={ListItemStories.ListItem}
+  exclude={['suffix', 'prefix', 'title', 'mode']}
```

Comment:
`exclude` allows you to remove argTypes from the props table that you don't want to show